android/app/src/main/res/layout/load.xml

```diff
@@ -48,8 +49,36 @@
             app:layout_constraintBottom_toBottomOf="@+id/etTransaction"
             app:layout_constraintEnd_toEndOf="@+id/textView4"
             app:layout_constraintTop_toTopOf="@+id/etTransaction"
-            app:srcCompat="@drawable/ic_paste_24dp"/>
+            app:srcCompat="@drawable/ic_paste_24dp" />
+
+        <androidx.constraintlayout.widget.Barrier
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            app:barrierDirection="end"
+            app:constraint_referenced_ids="idStatusLabel"
+            tools:layout_editor_absoluteX="411dp" />
+
+        <TextView
+            android:id="@+id/idStatusLabel"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="24dp"
+            android:text="@string/status"
+            android:visibility="visible"
+            app:layout_constraintBaseline_toBaselineOf="@+id/idTxStatus"
+            app:layout_constraintStart_toStartOf="parent" />
 
+        <TextView
+            android:id="@+id/idTxStatus"
+            style="@style/Medium"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="6dp"
+            android:layout_marginTop="6dp"
+            android:text="@string/empty"
+            android:visibility="visible"
+            app:layout_constraintStart_toEndOf="@+id/idStatusLabel"
+            app:layout_constraintTop_toBottomOf="@+id/etTransaction" />
```

Comment:
This field doesn't wrap correctly: 

<img src="https://user-images.githubusercontent.com/166954/121091114-67ebae80-c7e1-11eb-94c1-950cc76fd021.png" width="300"/>

Reply:
Thanks, yes, I'll look at them tomorrow.